### DISCUSSION TITLE
The answer to: "How to have images for both light and dark themes?"

### DISCUSSION THREAD

**Discussion by mcanouil** [quarto-dev/quarto-cli#5758] [Show and Tell]
### Description

You are using a light and a dark theme?
You are happy to be able,to switch between them by clicking on the toggle icon in the top right corner of the page?
But you still have the same images in both themes?
Then this is the right post for you!

https://mickael.canouil.fr/posts/2023-05-30-quarto-light-dark/

![featured](https://github.com/quarto-dev/quarto-cli/assets/8896044/3cbd4407-9d19-488b-b675-5191710eaca2)

**Comment by ute** [comment]
Another reason to use ggplot. If this also existed for base R... 😏
